google-marketplace/README.md

```diff
@@ -0,0 +1,173 @@
+```
+
+###### Clone this repository
+
+Clone this repository.
+
+```shell
+git clone git@github.com:DataDog/datadog-agent.git
+```
+
+###### Install the Application resource definition
+
+An Application resource is a collection of individual Kubernetes components,
+such as Services, Deployments, and so on, that you can manage as a group.
+
+To set up your cluster to understand Application resources, navigate to the
+`google-marketplace` folder in the repository, and run the following command:
+
+```shell
+make crd/install
+```
+
+You need to run this command once.
+
+The Application resource is defined by the
+[Kubernetes SIG-apps](https://github.com/kubernetes/community/tree/master/sig-apps)
+community. The source code can be found on
+[github.com/kubernetes-sigs/application](https://github.com/kubernetes-sigs/application).
+
+#### Install the Application
+
+Navigate to the `google-marketplace` folder:
+
+```shell
+cd datadog-agent/google-marketplace
+```
+
+##### Configure the app with environment variables
+
+Choose an instance name and
+[namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)
+for the app. In most cases, you can use the `default` namespace.
+
+```shell
+export namespace=default
+export name=datadog-agent
+```
+
+Configure the container image:
+
+```shell
+export datadogAgentImage=datadog/agent:latest
+```
+
+The image above is referenced by
+[tag](https://docs.docker.com/engine/reference/commandline/tag). We recommend
+that you pin each image to an immutable
+[content digest](https://docs.docker.com/registry/spec/api/#content-digests).
+This ensures that the installed application always uses the same images,
+until you are ready to upgrade.
+
+Fill in your Datadog API key:
+
+```shell
+export apiKeyEncoded=$(echo YOUR_DATADOG_API_KEY | base64)
+```
+
+##### Configure the service account
+
+The Datadog agent needs a service account in the target namespace with cluster wide
+permissions to inspect Kubernetes resources.
+
+Create a ClusterRole in your target Kubernetes cluster, a service account in the namespace specified earlier, and a ClusterRoleBinding to tie them together:
+
+```shell
+export serviceAccount=datadog-agent-sa
+make rbac/install  # from the datadog-agent/google-marketplace folder
+```
+
+##### Install the Datadog application
+
+Install the Datadog agent with the following command:
+
+```bash
+make app/install
+```
+
+This will expand the templates located in the `datadog-agent/google-marketplace/manifest` folder with the parameters given previously, and apply them, creating a Secret for the API key, and a DaemonSet tied to the service account previously created.
+
+## Basic Usage
+
+Once the application is installed, your Kubernetes nodes will show up in https://app.datadoghq.com/infrastructure/map and metrics will start flowing to your Datadog account!
+
+To further configure your agents, and setup your account, refer to [the documentation](https://docs.datadoghq.com/).
```

Comment:
r/`refer to [the documentation](https://docs.datadoghq.com/).`/`refer to [the Datadog documentation](https://docs.datadoghq.com/).`